### ISSUE TITLE
Hypocenter using 2 different 1D velocity models

### ISSUE THREAD

**Issue by Cthuulhaa** [ut-beg-texnet/NonLinLoc#58]
Dear Anthony,

I used NonLinLoc to locate an event in Austria using 2 1D velocity models: a simple 2-layer model, which is a mean velocity model for Germany, and a model for the Central Alps, Switzerland. I would say the uncertainties are in both cases reasonable because of the good station coverage, however the depths are quite different, so my question is at which information I should pay special attention to know which model "to trust"?

Location with 2-layer model:
![NLL_confidence_ellipsoid_2](https://github.com/user-attachments/assets/638c027f-2a8b-4d62-b926-831b4194176c)

Location with model for Central Alps:
![NLL_confidence_ellipsoid_11](https://github.com/user-attachments/assets/d1dc23d2-a68d-43b7-ad83-85a78ed24cbe)

Station coverage:
![epicenter_stations](https://github.com/user-attachments/assets/a78d2202-6f5e-4a3e-a115-43822425b4ec)

Cheers!

**Comment by alomax** [comment]
Hi @Cthuulhaa,

This is a key question, and one which I am still figuring out how best to understand and answer.

Currently, my comments would be that any arrive-time based location procedures (and any follow-on, differential timing relocations)  constrain mainly the relative location of the hypocenters, relative to the stations *given* the travel-time field. This constraint comes from the geometry of stations with available picks, the nominal and real error in the picks and the velocity model (travel-times). Location procedures do not directly address or know about absolute, geographic epicenter and depth "accuracy"; constraining these require, in addition to a good station distribution and high-quality picks, a very accurate initial velocity model and perhaps calibration with ground truth information (Husen & Hardebeck 2010).

So, in a nutshell, you should decide from independent information which model is "better" and prefer the depths in that model. However, one of your models puts the event at 25 km depth, which may be reasonable for earthquake seismicity in the area, while the other model has the event pdf "stuck" to the surface, which is likely only reasonable if the source is an event at the surface or very shallow depth, such as a quarry blast.

I am very interested in any insights you have on these issues!

Best regards,
Anthony

Husen, S., & Hardebeck, J. (2010). Earthquake location accuracy. Community Online Resource for Statistical Seismicity Analysis. [https://doi.org/10.5078/CORSSA-55815573](https://doi.org/10.5078/CORSSA-55815573)
